proxy/amux/mux_test.go

```diff
@@ -0,0 +1,124 @@
+package amux_test
+
+import (
+	"testing"
+	  "github.com/stretchr/testify/assert"
+	"github.com/enfabrica/enkit/proxy/amux"
+	"github.com/enfabrica/enkit/proxy/amux/amuxie"
```

Comment:
nit: gofmt, and separate stdlib imports from third-party

Reply:
done.